src/components/Scope/index.js

```diff
@@ -41,11 +42,18 @@ export const Scope = ({ children, filterPlaceholder }) => {
 	const name = "scopeSelector";
 	const [currentScope, defaultNodeState, getScope] = useScopeData();
 	const [{ show = false, nodeState, filter }, updateViewState] = useViewState(name);
+	usePreviousModified(
+		show,
+		current =>
```

Comment:
This function can be simplifed to `current => current && updateViewState('nodeState', {})`.